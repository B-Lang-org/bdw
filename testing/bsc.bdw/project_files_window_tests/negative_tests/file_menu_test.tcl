
# Creates a new project
handlers::open_project project.bspec 

# Compiles the specified file
handlers::compile_file

# Compiles the specified file without considering dependencies 
handlers::compile_file mult4/Mult2.bsv 

# 
handlers::set_search_paths mult4 

while {$BSPEC(FILES) == ""} {
        update
}

# Compiles the specified file considering dependencies 
handlers::compile_file mult4/Mult2.bsv -withdependencies

# Compiles the specified file considering dependencies 
handlers::compile_file mult4/Mult2.bsv -typeeecheck

# Compiles the specified file considering dependencies 
handlers::compile_file mult4/Mult2.bsv -withdeps -typ

# Compiles the specified file considering dependencies 
handlers::compile_file file -withdeps

# Closes the project 
handlers::close_project

# Exits the worksptation
exit
