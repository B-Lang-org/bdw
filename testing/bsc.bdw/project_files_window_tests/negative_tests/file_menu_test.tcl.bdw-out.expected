The file is not specified
The file 'mult4/Mult2.bsv' is not a project file.
Unknown option: -withdependencies
Unknown option: -typeeecheck
Unknown option: -typ
The file 'file' does not exist.
